src/MicrosoftTeams.ts

```diff
@@ -409,6 +414,12 @@ namespace microsoftTeams
 
     export namespace authentication
     {
+        let authWindow: Window;
+        let authInterval: number;
```

Comment:
Yup, that's what I am doing now. I still don't like the fact that there is a generic set of childWindow handling that lives in the microsoftTeams namespace and a further set of auth-specific logic that also applies to the child window and lives in the microsofTeams.authentication namespace. I will have to think about how we can clean this up in the future.